issue found on: analyser/aggregator.go:213:2: rangeValCopy

an issue has been detected in source code: ./analyser/aggregator.go:213:2: rangeValCopy: each iteration copies 144 bytes (consider pointers or indexing)

--- a/analyser/aggregator.go
+++ b/analyser/aggregator.go
@@ -209,9 +209,9 @@ func messageWithOffsetIn(entries []AggregatorLogEntry, offset int) bool {
 
 func diffEntryListsByOffset(list1, list2 []AggregatorLogEntry) []AggregatorLogEntry {
 	diff := []AggregatorLogEntry{}
-	for _, element := range list1 {
-		if !messageWithOffsetIn(list2, element.Offset) {
-			diff = append(diff, element)
+	for i := range list1 {
+		if !messageWithOffsetIn(list2, list1[i].Offset) {
+			diff = append(diff, list1[i])
 		}
 	}
 	return diff

